api/src/services/auth/group/group.schema.ts

```diff
@@ -35,6 +36,16 @@ export const groupSchema = Type.Object(
         },
       ),
     ),
+    role: Type.Optional(
```

Comment:
Well, I'm not sure if you have to name it `role` or `policy`.

If we decide to name this `policy`, maybe "all" need to be named `policy`:
* group schema (query ok, not the main schema)
* group model (ok)
* group resolver

Reply:
All right, I add it in the #231